{"schemaVersion":{"patch":0,"minor":3,"major":0},"metadata":{"role":"collectionGroup","roleHeading":"API Collection","modules":[{"name":"VernissageServer"}],"title":"LocalizedError Implementations"},"topicSections":[{"generated":true,"title":"Instance Properties","identifiers":["doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/failureReason","doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/helpAnchor","doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/recoverySuggestion"]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vernissageserver\/openaierror\/localizederror-implementations"]}],"kind":"article","sections":[],"identifier":{"url":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/VernissageServer\/documentation\/VernissageServer","doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError"]]},"references":{"doc://VernissageServer/documentation/VernissageServer/OpenAIError/recoverySuggestion":{"role":"symbol","type":"topic","title":"recoverySuggestion","kind":"symbol","url":"\/documentation\/vernissageserver\/openaierror\/recoverysuggestion","abstract":[],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"recoverySuggestion"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/recoverySuggestion"},"doc://VernissageServer/documentation/VernissageServer":{"identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer","kind":"symbol","type":"topic","abstract":[{"type":"text","text":"Application which is main API component for Vernissage photos sharing platform."}],"url":"\/documentation\/vernissageserver","role":"collection","title":"VernissageServer"},"doc://VernissageServer/documentation/VernissageServer/OpenAIError/failureReason":{"type":"topic","title":"failureReason","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"kind":"symbol","identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/failureReason","url":"\/documentation\/vernissageserver\/openaierror\/failurereason","abstract":[]},"doc://VernissageServer/documentation/VernissageServer/OpenAIError":{"navigatorTitle":[{"text":"OpenAIError","kind":"identifier"}],"kind":"symbol","role":"symbol","type":"topic","url":"\/documentation\/vernissageserver\/openaierror","identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError","abstract":[{"type":"text","text":"Errors returned during using OpenAI endpoints."}],"title":"OpenAIError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"OpenAIError"}]},"doc://VernissageServer/documentation/VernissageServer/OpenAIError/helpAnchor":{"role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}],"kind":"symbol","url":"\/documentation\/vernissageserver\/openaierror\/helpanchor","abstract":[],"title":"helpAnchor","identifier":"doc:\/\/VernissageServer\/documentation\/VernissageServer\/OpenAIError\/helpAnchor","type":"topic"}}}